recipe/conda_build_config.yaml

```diff
@@ -10,12 +21,3 @@ FINAL_CFLAGS:
 
 FINAL_CXXFLAGS:
   - -fvisibility-inlines-hidden -std=c++14 -fmessage-length=0 -march=k8 -mtune=haswell -ftree-vectorize -fstack-protector-strong -O2 -ffunction-sections -pipe
```

Comment:
Not sure if `FINAL_CFLAGS` / `FINAL_CXXFLAGS` is still relevant? I cannot find the corresponding settings in https://github.com/conda-forge/ctng-compilers-feedstock, as the comment above would imply.

Reply:
This moved to https://github.com/conda-forge/ctng-compiler-activation-feedstock/blob/main/recipe/conda_build_config.yaml; also needs to be updated...